### PR TITLE
tried mappo and tweaking env specs

### PR DESCRIPTION
**Pull request by kfu02**

### CODE REVIEW

File: jaxmarl/environments/mpe/simple_transport.py [hunk 1]
```diff
@@ -261,9 +261,9 @@ def reset(self, key: chex.PRNGKey) -> Tuple[chex.Array, State]:
         # randomly sample N_agents' capabilities from the possible agent pool (hence w/out replacement)
         selected_agents = jax.random.choice(key_a, self.agent_range, shape=(self.num_agents,), replace=False)
         
-        agent_rads = self.agent_rads[selected_agents]
-        agent_accels = self.agent_accels[selected_agents]
-        agent_capacities = self.agent_capacities[selected_agents]
+        agent_rads = jnp.array([0.2, 0.2, 0.2, 0.2]) # self.agent_rads[selected_agents]
```
Comment (shalin-jain) [inline jaxmarl/environments/mpe/simple_transport.py:264]: assuming this was a debugging artifact?

**Review by shalin-jain — APPROVED** [review]

File: jaxmarl/environments/mpe/simple_transport.py (hunk 1 above)
Comment (kfu02) [reply jaxmarl/environments/mpe/simple_transport.py:264]: actually no, the MAPPO pipeline has no way of passing in a separate environment YAML, so I had to hardcode in the capabilities here

Comment (kfu02) [reply jaxmarl/environments/mpe/simple_transport.py:264]: I take this back, there is no way to load the separate env YAML but there is an ENV_KWARGS param in the mappo script that I missed completely

Comment (kfu02) [reply jaxmarl/environments/mpe/simple_transport.py:264]: (so you can undo this change)

**Pull request merged**